### DISCUSSION TITLE
Class name of City Environ

### DISCUSSION THREAD

**Discussion by huaihailv** [microsoft/AirSim#4508] [Support Q&A]
In order to detect objects specifically for pedestrian and car, I've already listed the objects by simListSceneObjects.
But I can't find the class according to the pedestrian and bicycle, please tell me the straight answer or a solution to get it
![微信图片_20220429205238](https://user-images.githubusercontent.com/77001968/165947866-63a2523a-9616-4abf-ac71-74d93692e77c.png)
.

**Comment by zimmy87** [comment]
With how the city environment is implemented currently, the pedestrians and cars can't be processed are regular scene objects. See the discussion in #4196 for more details. Basically, pedestrians and cars are part of instanced static meshes, so there is not a 1:1 correspondence between those objects and engine actors. In order to make the object detection APIs work on these objects, you would need to modify the APIs to iterate through individual instances inside an instanced static mesh.
